### PR TITLE
Fix: Kits Defaults - Support for inner section [ED-8585]

### PR DESCRIPTION
**Pull request by Nevoss**
- [x] Missing FE tests

### CODE REVIEW

File: modules/kit-elements-defaults/module.php [hunk 1]
```diff
@@ -58,6 +58,7 @@ public function __construct() {
 
 		if ( is_admin() ) {
 			( new Import_Export() )->register();
+			( new Import_Export() )->register();
```
Comment (StyleShit) [inline modules/kit-elements-defaults/module.php:61]: remove

File: tests/jest/unit/modules/kit-elements-defaults/assets/js/editor/hooks/data/fill-defaults-on-drop.test.js [hunk 2]
```diff
@@ -92,6 +92,33 @@ describe( `$e.run('preview/drop') - Hook: FillDefaultsOnDrop`, () => {
...
+				model: {
+					elType,
+					widgetType,
+					isInner,
+					settings: {},
+				},
+			} );
+
+			// Assert
+			expect( getElementDefaults ).toHaveBeenNthCalledWith( 1, expectedType );
```
Comment (StyleShit) [inline tests/jest/unit/modules/kit-elements-defaults/assets/js/editor/hooks/data/fill-defaults-on-drop.test.js:118]: why here you use `Nth` and in the `create` test you don't?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: modules/kit-elements-defaults/module.php (hunk 1 above)
Comment (Nevoss) [reply modules/kit-elements-defaults/module.php:61]: opss

Comment (Nevoss) [reply modules/kit-elements-defaults/module.php:61]: Done

Comment (Nevoss) [reply modules/kit-elements-defaults/module.php:61]: Opss, done

File: tests/jest/unit/modules/kit-elements-defaults/assets/js/editor/hooks/data/fill-defaults-on-drop.test.js (hunk 2 above)
Comment (Nevoss) [reply tests/jest/unit/modules/kit-elements-defaults/assets/js/editor/hooks/data/fill-defaults-on-drop.test.js:118]: Done

**Review by StyleShit — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**